test/GRAPHFILES

```diff
@@ -322,3 +322,5 @@ statements/lydia/moduleVersusFunction.graph
 # suite: Samples (bogus)
 Samples/Performance/foo.graph
 Samples/Performance/cleanCopy/foobar.graph
+# suite: Sample Radix Sort
+library/packages/Sort/RadixSort/radixsortMSB.graph
```

Comment:
Despite not being added to the standard library yet, I think we should add this test to the standard library suite with the other sort performance tests (`# suite: Standard Library`)